.gitignore

```diff
@@ -15,3 +15,5 @@ nanocat
 perf
 .libs
 Makefile
+.idea
+*.iml
```

Comment:
What are .idea and .iml files?  Maybe a better place for these is `~/.gitignore_global`?